css/styles.css

```
@@ -0,0 +1,62 @@
+
+.box {
+    color: black;
+    width: 100px;
+    height: 100px;
+    border: 1px solid black;
+    display: inline-block;
+    margin: -2px;
+    align-content: center;
+}
+
+.black {
+    background-color: black;
+    width: 100px;
+    height: 100px;
+    border: 1px solid black;
+    display: inline-block;
+    margin: -2px;
+    align-content: center;
+}

```

у тебя блоки .box и .black повторяются поэтому в .box можно оставить всё что есть и в .black дописать пару недостающих рулов
```suggestion
.black {
    background-color: black;
}

```